### DISCUSSION TITLE
[Mobile] selection in readonly mode

### DISCUSSION THREAD

**Discussion by WingCH** [singerdmx/flutter-quill#165] [Mobile]
I have tried running `example` directory successfully before creating an issue

in read_only_page.dart, set showCursor to false

```dart
...
    var quillEditor = QuillEditor(
      controller: controller!,
      scrollController: ScrollController(),
      scrollable: true,
      focusNode: _focusNode,
      autoFocus: true,
      readOnly: !_edit,
      expands: false,
      padding: EdgeInsets.zero,
      showCursor: false, // add this
    );
...
```

Step 1: select text
Step 2: click copy

"selection point" should not show
![image](https://user-images.githubusercontent.com/19588517/113655136-d91cc180-96cb-11eb-8323-95ff5dc79d3c.png) 

https://user-images.githubusercontent.com/19588517/113655020-a8d52300-96cb-11eb-9589-46c2dd48522a.mp4

**Comment by singerdmx** [comment]
why did you `set showCursor to false` ? 
I am unsure it is a good combo

**Comment by WingCH** [comment]
Thanks for your quick reply.
because i don't want show `Cursor` in read mode

![image](https://user-images.githubusercontent.com/19588517/113664090-ba272b00-96dd-11eb-9d07-54f4da8700b1.png)

**Comment by singerdmx** [comment]
I can repro this. You are welcome to submit a pull request to address it

**Comment by WingCH** [comment]
Sorry, I have no idea about this Issue...🤧
